server/planning/common.py

```diff
@@ -425,19 +428,29 @@ def get_version_item_for_post(item):
 
 
 @celery.task(soft_time_limit=600)
```

Comment:
Now that I think about it, does this need to use celery anymore?
With the old superdesk-core code/content, we changed from using celery task for enqueue to an async function. We might be able to do the same thing here

Reply:
If you're able to convert this to async, don't forget the calling convention, currently using `apply_async` because it's a celery task

Reply:
done